components/admin/requests/Header.tsx

```diff
@@ -1,120 +1,40 @@
 import ShopifyBadge from '@components/admin/ShopifyBadge';
-import { ApplicationStatus, ApplicationType } from '@lib/graphql/types';
-import ProcessingTasksFooter from './processing/ProcessingFooter';
+import PermitTypeBadge from '@components/admin/PermitTypeBadge';
+import { ApplicationStatus, ApplicationType, PermitType } from '@lib/graphql/types';
 
 type RequestHeaderProps = {
-  readonly applicationId: number;
-  readonly applicationStatus?: ApplicationStatus;
   readonly applicationType: ApplicationType;
+  readonly permitType: PermitType;
   readonly createdAt: Date;
-  readonly allStepsCompleted: boolean;
-  readonly applicantId?: number;
+  readonly applicationStatus?: ApplicationStatus;
   readonly paidThroughShopify?: boolean;
   readonly shopifyOrderID?: string;
   readonly shopifyOrderNumber?: string;
 };
 
 /**
  * Header of View Request page
+ * @param applicationType Type of application
+ * @param permitType Type of permit
+ * @param createdAt Date permit was created at
  * @param applicationStatus Status of application
- * @param createdAt Date of application creation
- * @param allStepsCompleted Whether all processing tasks are complete
+ * @param paidThroughShopify If the permit fee was paid through Shopify
+ * @param shopifyOrderID Order ID of Shopify payment if paid through Shopify
+ * @param shopifyOrderNumber Order number of Shopify payment if paid through Shopify
  */
 export default function RequestHeader({
-  applicationId,
-  applicationStatus,
-  createdAt,
-  allStepsCompleted,
   applicationType,
-  applicantId,
+  permitType,
+  createdAt,
+  applicationStatus,
   paidThroughShopify,
   shopifyOrderID,
   shopifyOrderNumber,
 }: RequestHeaderProps) {
-  /**
-   * Returns the appropriate header button(s) to be displayed depending on the current application status
-   * @returns Rendered button component(s) or null.
-   */
-  const _renderActionButtons = () => {
-    switch (applicationStatus) {
-      case 'PENDING':
-        return (
-          <HStack spacing={3}>
-            <RejectRequestModal applicationId={applicationId}>
-              <Button bg="secondary.critical" _hover={{ bg: 'secondary.criticalHover' }}>
-                Reject
-              </Button>
-            </RejectRequestModal>
-            <ApproveRequestModal applicationId={applicationId}>
-              <Button>Approve</Button>
-            </ApproveRequestModal>
-          </HStack>
-        );
-      case 'IN_PROGRESS':
-        return (
-          <ProcessingTasksFooter
-            applicationId={applicationId}
-            applicantId={applicantId}
-            allStepsCompleted={allStepsCompleted}
-          />
-        );
-      default:
-        return null;
-    }
-  };
-
-  /**
-   * Returns the appropriate 'More Actions' dropdown to be displayed depending on the current application status
-   * @returns Rendered 'More Actions' dropdown component or null
-   */
-  const _renderMoreActionsDropdown = () => {
```

Comment:
Removed the "more actions" dropdown that was there before since it's not on the designs anymore